frontend/src/v4/modules/viewerGui/viewerGui.redux.ts

```diff
@@ -44,11 +45,14 @@ export const { Types: ViewerGuiTypes, Creators: ViewerGuiActions } = createActio
 	decreaseHelicopterSpeed: ['teamspace', 'modelId'],
 	setIsFocusMode: ['isFocusMode'],
 	setClippingMode: ['mode'],
-	setClippingModeSuccess: ['mode'],
+	updateClipModeSuccess: ['mode'],
 	setClipEdit: ['isClipEdit'],
-	setClipEditSuccess: ['isClipEdit'],
+	updateClipEditSuccess: ['isClipEdit'],
 	setClipNumber: ['clipNumber'],
-	updateClipState: ['clipNumber'],
+	setGizmoMode: ['mode'],
+	setGizmoModeSuccess: ['mode'],
+	updateClipMode: ['clipMode'],
+	updateClipEdit: ['clipEdit'],
```

Comment:
The update methods just update the frontend state whereas the set methods change the state of the actual viewer. This was done because I needed away to listen to the viewer and update the frontend state accordingly.

Nevermind, I just tried again and found a way of making it work with just one method for each